test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityFirstReturnControllerSpec.scala

```diff
@@ -1839,3 +1847,30 @@ class YearToDateLiabilityFirstReturnControllerSpec
     }
 
 }
+
+object YearToDateLiabilityFirstReturnControllerSpec extends Matchers {
+  def validateYearToDateLiabilityFirstReturnPage(
+    completeYearToDateLiabilityAnswers: CompleteYearToDateLiabilityAnswers,
+    doc: Document
+  )(implicit messages: MessagesApi, lang: Lang): Unit = {
+    doc.select("#estimatedIncome-value-answer").text() shouldBe formatAmountOfMoneyWithPoundSign(
+      completeYearToDateLiabilityAnswers.estimatedIncome.inPounds()
+    )
+
+    if (completeYearToDateLiabilityAnswers.personalAllowance.isDefined)
```

Comment:
you can use `.foreach` on the optional `.personaAllowace` to avoid doing `.getOrElse(sys.error(...))`